testfiles/ASHRAE9012016_ApartmentMidRise_Denver.idf

```diff
@@ -5586,9 +5586,9 @@
     1,                       !- Multiplier
     4,                       !- Number of Vertices
     46.327300000000,0.381000000000,2.173100000000,  !- X,Y,Z ==> Vertex 1 {m}
-    46.327300000000,0.381000000000,0.954000000000,  !- X,Y,Z ==> Vertex 2 {m}
+    46.327300000000,1.295300000000,2.173100000000,  !- X,Y,Z ==> Vertex 2 {m}
     46.327300000000,1.295300000000,0.954000000000,  !- X,Y,Z ==> Vertex 3 {m}
-    46.327300000000,1.295300000000,2.173100000000;  !- X,Y,Z ==> Vertex 4 {m}
+    46.327300000000,0.381000000000,0.954000000000;  !- X,Y,Z ==> Vertex 3 {m}
```

Comment:
Now that's interesting.  This worked in an older version of the file, but I guess things got more strict about the surface direction along the way.  Good find.

Reply:
Yeah, I think these files were translated starting from version 8.6.